momepy/functional/_dimension.py

```diff
@@ -11,7 +14,10 @@
 ]
 
 
-def volume(area, height):
+def volume(
+    area: NDArray[np.float_] | Series,
```

Comment:
Maybe we can do with a bare `NDArray`? Since I don't know how to add the same to Series anyway.

Reply:
Did you try: `Series[np.float_]`